test/test_app.py

```diff
@@ -2,13 +2,19 @@
 from app import app
 
 
-class BasicTestCase(unittest.TestCase):
+class BasicTests(unittest.TestCase):
     def test_index(self):
         tester = app.test_client(self)
         response = tester.get('/', content_type='html/text')
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.data, b'Hello World!')
 
+    def test_json(self):
```

Comment:
Maybe is better to split these into two test classes, such that we can put together all tests that belong to a certain endpoint (eg `/EUS` or `/`).

Reply:
done